packages/wagmi/src/client.ts

```diff
@@ -377,6 +379,26 @@ export class Web3Modal extends Web3ModalScaffold {
     })
 
     this.setEIP6963Enabled(w3mOptions.enableEIP6963 !== false)
+
+    this.subscribeShouldUpdateToAddress((newAddress?: string) => {
```

Comment:
Is this something we need to implement for other clients (ethers/ethers5)?

Reply:
was already implemented, I added this cause it was missing 🖖